chain/client/src/client_actor.rs

```diff
@@ -255,6 +255,7 @@ impl Handler<NetworkClientMessages> for ClientActor {
 
     #[perf_with_debug]
     fn handle(&mut self, msg: NetworkClientMessages, ctx: &mut Context<Self>) -> Self::Result {
+        let _span = debug_span!(target: "client", "NetworkClientMessages").entered();
```

Comment:
If we are touching many of the lines anyway, maybe move to qualified `tracing::debug_span!` as well?